Update README.md (#228)
redpanda-data/helm-charts: charts/redpanda/README.md

[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/redpanda-data)](https://artifacthub.io/packages/search?repo=redpanda-data)

The Redpanda Helm chart deploys a Redpanda cluster in Kubernetes, and provides the following features:

- Schema registry (enabled by default)
- REST (aka PandaProxy, enabled by default)
- TLS
- SASL
- External access

## Requirements

- Helm version 3.6.0 or later
- Kubernetes version 1.21.0 or later

> - Cert-manager 1.9.0 or later (only if you want TLS support)

Reviewer: "if you want" sounds wishy-washy.

```suggestion
- Cert-manager 1.9.0 or later (required for TLS support only)
```

## Installation

To get started, see the [Redpanda documentation][kubernetes-qs-dev].

For examples of using the Helm chart, see the [`examples/` directory][examples]. Each example focuses on a specific feature.

## Configuration

> The Redpanda Helm chart is configured in the [`values.yaml`][values] file. You can override the default values in this file to customize your deployment. For example, you can do the following:

Reviewer: Though technically true, this leads novice users into thinking that they need to modify this file. What is the goal of this line? Maybe we can think of better phrasing.

Author: We link to the Helm docs for details about overriding the defaults.

Maybe we can be explicit and say `To customize your deployment, you can override the default values in your own YAML file or in the command line by using the `--set` option.

Reviewer: Yes, that sounds much better.

## Requirements

The [values.yaml][values] file is documented throughout.
Please see this file for more details.
- Helm version 3.6.0 or later
- Kubernetes version 1.21.0 or later
- Cert-manager 1.9.0 or later (only if you want TLS support)

## Installation

See the [60-Second Guide for Kubernetes][kubernetes-qs-dev]
To get started, see the [Redpanda documentation][kubernetes-qs-dev].

For examples of using the Helm chart, see the [`examples/` directory][examples]. Each example focuses on a specific feature.

## Configuration

The Redpanda Helm chart is configured in the [`values.yaml`][values] file. You can override the default values in this file to customize your deployment. For example, you can do the following:

- Specify which Kubernetes components to deploy.

- Configure the deployed Kubernetes components.

To learn how to override the default values in the `values.yaml` file, see the [Helm documentation](helm).

All configuration options for the Redpanda Helm chart are documented in the [`values.yaml`][values] file.

## Contributing

> If you have improvements that can be made to this Helm chart, please consider becoming a contributor.

Reviewer: This is a call to action. Let's keep it.

> To contribute to the Helm chart, see our [contribution guidelines][contributing].

Author: ```suggestion
If you have improvements that can be made to this Helm chart, please consider becoming a contributor. To contribute to the Helm chart, see our [contribution guidelines][contributing].
```
